CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
 ### Updates
 
 - Updates storybook URL from /reservoir/v0_26 to /reservoir/v0.
+- Updates the `HelperErrorText` styling to correctly display when used with a `Select` element with a `labelPosition` of inline.
```

Comment:
This needs to be moved up under the "## Prerelease" section. This is under the already released 0.26.1 version. When you merge in the `development` branch, you'll see that there are now new additions in the CHANGELOG and you can add this to the proper area.